pkg/ipam/node.go

```diff
@@ -828,7 +831,7 @@ func (n *Node) update(origNode, node *v2.CiliumNode, attempts int, status bool)
 //
 // For now, this function is only needed in ENI IPAM mode. In the future, this
 // may also be needed for Azure IPAM as well.
-func adjustPreAllocateIfNeeded(node *v2.CiliumNode) {
+func adjustPreAllocateIfNeeded(node *v2.CiliumNode) (err error) {
```

Comment:
Nit: no need to name return values; we only have one

Reply:
I reverted this change as well 👍